src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -467,98 +461,29 @@ export type ModifyQueryParams = {
 /* fields and methods in the class below are ordered based on roughly
 /* chronological setup concerns, rather than on encapsulation and public API */
 /* tslint:disable: member-ordering */
-export class ResultsViewPageStore {
+export class ResultsViewPageStore
+    implements
```

Comment:
it seems strange to me that we would make the store implement these rather than just have a single node of the store which is of type IDriverSettingsProps.  ditto for the rest of these.

Reply:
Well, that depends on whether the implementation of the interface is the same and can be located in a shared class. This is not the case for most members (compare _ResultsViewPageStore_ with _ComparisonStore_ f.i. for customDriverAnnotationReport, includeGermlineMutations, etc.). In my view these classes implementing these interfaces is perfectly fine.